site/owid.entry.ts

```diff
@@ -58,9 +58,13 @@ window.runSiteFooterScripts = () => {
     runCookiePreferencesManager()
     runCovid()
     runFootnotes()
+    renderProminentLink()
```

Comment:
Seems like both branches below call this function anyway?

Reply:
perfect -- fixed